docs/guidelines/templates/templates-type.md

```diff
@@ -22,7 +22,7 @@ This table lists all document types used to fully describe a Kyma area and/or co
 | [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`architecture.md`](https://github.com/kyma-project/community/blob/main/templates/resources/architecture.md) | Use the template to describe the architecture of a Kyma component. | All contributors |
 | [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`configuration.md`](https://github.com/kyma-project/community/blob/main/templates/resources/configuration.md) | Use the template to describe configuration options for a Kyma chart or sub-chart. | All contributors |
 | [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`custom-resource.md`](https://github.com/kyma-project/community/blob/main/templates/resources/custom-resource.md) | Use the template to describe a custom resource (CR) and its elements. | All contributors |
-| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`tutorials.md`](https://github.com/kyma-project/community/blob/main/templates/resources/tutorial.md) | Use the template to provide a clear step-by-step instruction that helps the user to understand a given concept better. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`tutorials.md`](https://github.com/kyma-project/community/blob/main/templates/resources/task.md) | Use the template to provide a clear step-by-step instruction that helps the user to understand a given concept better. | All contributors |
```

Comment:
```suggestion
| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`task.md`](https://github.com/kyma-project/community/blob/main/templates/resources/task.md) | Use the template to provide a clear step-by-step instruction that helps the user to understand a given task better. | All contributors |
```

Reply:
I wouldn't use "concept" in the description. I've replaced it with "task, but it could be "process" or "procedure".

Reply:
Good catch! 
Thinking about it, we could probably stop the sentence after "instruction" :D